### PR TITLE
Site dev uncompress

### PR DESCRIPTION
**Pull request by paranoidjk**
wait for https://github.com/benjycui/bisheng/pull/99

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/1761)

### CODE REVIEW

File: site/bisheng.common.config.js [hunk 1]
```diff
@@ -36,4 +36,7 @@ module.exports = {
     ]);
     return config;
   },
+  htmlTemplateExtraData: {
+    isDev: process.env.NODE_ENV === 'development',
+  },
```
Comment (afc163) [inline site/bisheng.common.config.js]: No need this, just change `resolve.alias` according to `process.env.NODE_ENV`.

File: site/kitchen/src/static/template.html [hunk 2]
```diff
@@ -55,9 +55,12 @@
   </head>
   <body>
     <div id="react-content"></div>
+    {% if isDev %}
+    <script src="https://as.alipayobjects.com/g/component/??react/15.3.2/react.js,react/15.3.2/react-dom.js"></script>
```
Comment (afc163) [inline site/kitchen/src/static/template.html]: 开发环境下不用引 external，直接走 webpack。

Comment (paranoidjk) [reply site/kitchen/src/static/template.html]: 模板里面还是要有变量能判断，决定是否引入 scripts

Comment (paranoidjk) [reply site/kitchen/src/static/template.html]: 开发环境也 external 的好处是可以加快构建速度，调试的时候 dev server 快一点

File: site/desktop/src/static/template.html [hunk 3]
```diff
@@ -99,9 +99,11 @@
   </head>
   <body>
     <div id="react-content"></div>
+    {% if isDev %}
+    <script src="https://as.alipayobjects.com/g/component/??react/15.3.2/react.js,react/15.3.2/react-dom.js"></script>
```
Comment (afc163) [inline site/desktop/src/static/template.html]: https://as.alipayobjects.com/g/component/??react/15.6.1/react.js,react/15.6.1/react-dom.js

Comment (paranoidjk) [reply site/desktop/src/static/template.html]: 稍后我把 version 改成从 pkg.version 读变量。

**Pull request merged**